server/src/config.js

```diff
@@ -133,7 +133,7 @@ var conf = convict({
   // This is mostly configurable for debugging purposes:
   checkDeletedInterval: {
     doc: "Frequency in seconds to check for items that should be purged",
-    format: Number,
+    format: "int",
```

Comment:
Looks like you are reverting a change that's landed in master. Probably need to fetch master and rebase.